plugins/catalog/api-report.md

```diff
@@ -337,6 +337,14 @@ export interface DependsOnResourcesCardProps {
 // @public
 export const EntityAboutCard: (props: AboutCardProps) => JSX.Element;
 
+// Warning: (ae-forgotten-export) The symbol "EntityContextMenuProps" needs to be exported by the entry point index.d.ts
+// Warning: (ae-missing-release-tag) "EntityContextMenu" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
sorry can we resolve these two warnings? Ideally `EntityContextMenu` should become something like:

```ts
/**
 * @public
 */
export function EntityContextMenu(props: {
  UNSTABLE_extraContextMenuItems?: ExtraContextMenuItem[];
  UNSTABLE_contextMenuOptions?: UnregisterEntityOptions;
  onUnregisterEntity: () => void;
  onInspectEntity: () => void;
}) {
```

getting rid of `EntityContextMenuProps` completely.

Reply:
I was able to remove these warnings above, but now I'm seeing two new warnings starting on Line 634 in [the same file](plugins/catalog/api-report.md):

```ts
 // Warnings were encountered during analysis:
 //
 // src/components/EntityContextMenu/EntityContextMenu.d.ts:13:5 - (ae-forgotten-export) The symbol "ExtraContextMenuItem" needs to be exported by the entry point index.d.ts
 // src/components/EntityContextMenu/EntityContextMenu.d.ts:14:5 - (ae-forgotten-export) The symbol "UnregisterEntityOptions" needs to be exported by the entry point index.d.ts
```

There's an existing note about the types in question on Line 51 in [EntityContextMenu.tsx](plugins/catalog/src/components/EntityContextMenu/EntityContextMenu.tsx):

```ts
// NOTE(freben): Intentionally not exported at this point, since it's part of
// the unstable extra context menu items concept below
interface ExtraContextMenuItem {
  title: string;
  Icon: IconComponent;
   onClick: () => void;
 }
```

Would it be better to include these definitions inline as well, or ignore the new warnings?

(FWIW - I see a duplicate definition of `ExtraContextMenuItem` in [EntityLayout.tsx](plugins/catalog/src/components/EntityLayout/EntityLayout.tsx) on Line 145.)

Reply:
Referring back to [this comment](https://github.com/backstage/backstage/pull/21525#discussion_r1409805645) I've gone ahead and define it completely inline. Please let me know if I should make any other changes~